config/initializers/active_admin.rb

```diff
@@ -120,7 +120,7 @@
   # will call the method to return the path.
   #
   # Default:
-  config.logout_link_path = :destroy_admin_user_session_path
+  config.logout_link_path = :destroy_user_session_path
```

Comment:
Is `def destroy_admin_user_session_path` in app/helpers/application_helper.rb no longer needed?

Reply:
I guess so! I don't see it in here any more, I just have removed it some point.